server/src/utils/media.ts

```diff
@@ -259,7 +259,7 @@ export class BaseHWConfig extends BaseConfig implements VideoCodecHWConfig {
   }
 
   getSupportedCodecs() {
-    return [VideoCodec.H264, VideoCodec.HEVC, VideoCodec.VP9];
+    return [VideoCodec.H264, VideoCodec.HEVC, VideoCodec.VP9, VideoCodec.AV1];
```

Comment:
If you're changing this, you should override it in VAAPI since it doesn't support AV1 until FFmpeg 6.1.

Reply:
Got it, done

Reply:
Oh, and NVENC overrides this since it doesn't support VP9. I think it'd be cleaner to change the base method to only include H.264 and HEVC and let the others add onto that depending on what else they support.

Reply:
Is this in the scope of this PR or should I leave it like this for now?

Reply:
Since we're touching this code anyway, we may as well change it here.

Reply:
Done - I also removed AV1 since it's only implemented for SW right now. Can be added back as soon as the HW configs are extended to enable AV1.